domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -78,6 +79,7 @@
     private final YesNoOption paperResponse;
     private final LocalDateTime dateReferredForDirections;
     private final String preferredDQCourt;
+    private final UUID bulkPrintLetterId;
```

Comment:
For other IDs we're keeping them as strings, and the use of UUIDs is incidental. Would recommend continuing this practice.

Reply:
changed all UUID to string